doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/02_V4_to_V5.md

```diff
@@ -296,3 +296,23 @@ $this->get(\Pimcore\Cache\Runtime::class) // service container
 ## E-Commerce Framework
 
 For migration of E-Commerce-Framework have a look at the [breaking changes section](./03_Ecommerce_Framework/README.md).
+
+## Newsletter Source Adapter
+
+Newsletter Source Adapter implementation has been changed to be able to register them using configuration. It is very easy to migrate your Adapter to Pimcore 5.
+Since nothing really changed, you only need to create a factory service that creates your AddressSourceAdapter
+ - If you are using a very simple Adapter Source, you can use the DefaultFactory
+  ```yml
+  app.document.newsletter.factory.myAdapter:
+      class: Pimcore\Document\Newsletter\DefaultAddressSourceAdapterFactory
+      arguments:
+        - 'App\Document\Newsletter\AddressSourceAdapter\MyAdapter'
+  ```
+- If you are using a more complex Source Adapter, you can create your own Factory by implementing the interface Pimcore\Document\Newsletter\Factory\AddressSourceAdapterFactoryInterface
+- Add your Adapter Factory to the configuration:
+
+```yml
+pimcore:
+    newsletter:
```

Comment:
It's currently `pimcore.documents.newsletter` but for me `pimcore.newsletter` sounds better/more concise. Could you update the config + extension accordingly?